discovery-provider/src/queries/query_helpers.py

```diff
@@ -432,17 +432,6 @@ def populate_track_metadata(session, track_ids, tracks, current_user_id):
         else:
             track[response_name_constants.remix_of] = None
 
-        # Populate the remix_of tracks w/ the parent track's user and if that user saved/reposted the child
```

Comment:
this was a merge conflict that somehow got merged. Code is duplicated from above.